lib/companies/pending_changes.ex

```diff
@@ -71,6 +71,19 @@ defmodule Companies.PendingChanges do
     end
   end
 
+  def get_pending_changes_for(record) do
+    resource_name = struct_to_string(record)
+    id = record.id |> to_string()
```

Comment:
Since there's only one pipe here, what would you think about using the conventional function syntax?
```ex
id = to_string(record.id)
```

Reply:
This is the preferred way given the styleguide we try to follow. We may need to update our Credo (or add it) rules for some of these.

Additionally I'd likely capture the `record_id` value via a match and avoid dot notation (I don't know why, I don't much care for it personally).